### ISSUE TITLE
[Bug] `element-newline` incorrectly reports on siblings

### ISSUE THREAD

**Issue by RobertAKARobin** [yeonjuan/html-eslint#210]
For example, this test fails. It should pass because `b` and `c` are skipped:

```js
    {
      code: `<a>
<b></b><c></c>
</a>`,
      options: [
        {
          skip: ["b", "c"]
        }
      ]
    }
```

**Comment by RobertAKARobin** [comment]
Actually this seems to be a bug with checking sibling elements in general. Updated the main post.

**Comment by yeonjuan** [comment]
Hi @RobertAKARobin 
> This rule has an object option:
> "skip": skips newline checking for the specified element's children

This is the intended behavior: the `skip` option skips the occurrence of child elements of the specified tag.

**Comment by RobertAKARobin** [comment]
This additional functionality added in https://github.com/yeonjuan/html-eslint/pull/214
